src/components/Login/Login.jsx

```diff
@@ -56,15 +80,18 @@ function Login() {
 								Password
 								<input
 									className={styles.btnPass}
-									id="password"
-									placeholder="Password"
-									type="password"
+									name="password"
 									{...register('password', {
 										required: {
 											message: 'La contraseña es obligatoria',
 											value: true,
 										},
 									})}
+									// eslint-disable-next-line react/jsx-no-bind
```

Comment:
Seems that I put wrong the rule in eslint, please dissable react/jsx-no-bind from there, and remove this comment x2